City-of-Helsinki/outdoors-sports-map
# HULK-5 - Accessible colors for unusable units

## Description
Replace red with black. Applies to CSS colors and SVG/PNG icons.

**NOTE** Someone with better visual impairment knowledge needs to verify if difference between unusable and unknown is enough. Unknown status is using grey so if the contrast is not enough, it can be confused with unusable.

## Context
People with red-green color blindness have hard time separating good (green) and unusable (red) units. To avoid the confusion, unusable units/routes should be marked with black color.

[HULK-5](https://helsinkisolutionoffice.atlassian.net/browse/HULK-5)

## How Has This Been Tested?
Manual testing. No unit tests.

## Manual Testing Instructions for Reviewers
Open the map and verify that unused markers/routes are black. Also, verify that the status in the unit details view is using white text on black background.

## Screenshots
![Screenshot 2022-07-20 at 7 59 48](https://user-images.githubusercontent.com/14893875/179900620-39849ddd-8f0c-477a-8f98-f1e0dda40f68.png)
![Screenshot 2022-07-20 at 8 00 17](https://user-images.githubusercontent.com/14893875/179900629-ff231827-a890-44bc-b5c3-a0dcb6b4f157.png)
